### PR TITLE
test: raise timeout to avoid flaky test failures

### PR DESCRIPTION
**Pull request by danielnorberg**
Default timeout is 10 seconds. Raise timeout to 30 seconds.

Have less failures like this: https://circleci.com/gh/spotify/styx/1578

### CODE REVIEW

File: styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java [hunk 1]
```diff
@@ -286,31 +287,31 @@ long timeOffsetSeconds(int secondsOffset) {
...
   }
 
   void awaitNumberOfDockerRuns(int n) {
-    await().until(() -> dockerRuns.size() == n);
+    await().atMost(30, SECONDS).until(() -> dockerRuns.size() == n);
   }
 
   void awaitNumberOfDockerRunsWontChange(int n) {
-    await().pollDelay(1, TimeUnit.SECONDS).until(() -> {
+    await().pollDelay(1, SECONDS).until(() -> {
```
Comment (duchuyctlk) [inline styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java]: I could be `await().pollDelay(1, SECONDS).until(() -> dockerRuns.size() == n);`

Comment (danielnorberg) [inline styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java]: Indeed. Method is unused even so just removed it.

**Comment by honnix** [comment]
👍

**Comment by TinaRanic** [comment]
👍

**Pull request merged**